coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html

```diff
@@ -187,11 +220,50 @@ <h3 class="explorer-title">Featured Courses</h3>
       },
 
       _onCoursesResponse: function (event) {
-        this.set('courses', event.detail.response.data.allCourses);
-        this.set(
-          'filterOptions.status', 'all'); // Todo:  this._enrolledInAny() ? 'enrolled' : 'all'
+        var courses = event.detail.response.data.allCourses;
+        this.set('courses', courses);
+        this.set('filterOptions.status', 'all');
         this.set('categories', this._getCategories().sort());
         this.set('coursesLoaded', true);
+
+        var registeredCourseIds = [];
+        courses.edges.forEach(function (course) {
+          if (course.node.enrollment.enrolled === true) {
+            registeredCourseIds.push(course.node.id)
+          }
+        });
+        if (registeredCourseIds.length > 0) {
+          this.set('registeredCourses', registeredCourseIds);
+          this.set('loadDashboard', true);
+        } else {
+          this.set('dashboardLoaded', true);
```

Comment:
@andela-angene Is this right?

Reply:
@chegejames yes it it. On the student-dashboard, it stops the spinner and displays a message to the user that he/she has no registered courses.

Reply:
@andela-angene https://gmkr.io/s/59c9f0b6b4e651b93b47e459/0

Reply:
Already fixed @chegejames